docs/sphinx/contributing/documentation.rst

```diff
@@ -24,17 +24,17 @@ To build the |hpx| documentation, you need recent versions of the following
 packages:
 
 - ``python3``
-- ``sphinx 3.5.4`` (Python package)
-- ``sphinx_rtd_theme`` (Python package)
-- ``breathe 4.16.0`` (Python package)
+- ``sphinx 4.5.0`` (Python package)
+- ``sphinx-book-theme`` (Python package)
+- ``breathe 4.33.1`` (Python package)
 - ``doxygen``
 
 If the |python|_ dependencies are not available through your system package
 manager, you can install them using the Python package manager ``pip``:
 
 .. code-block:: bash
 
-   pip install --user sphinx sphinx_rtd_theme breathe
+   pip install --user sphinx sphinx-book-theme breathe
```

Comment:
pip install "sphinx<5" sphinx-book-theme breathe sphinxcontrib-bibtex

I needed to install sphinxcontrib-bibtex as well otherwise I had an error

Could not import extension sphinxcontrib.bibtex (exception: No module named 'sphinxcontrib.bibtex')